atom/browser/native_window.h

```diff
@@ -96,6 +96,9 @@ class NativeWindow : public brightray::DefaultWebContentsDelegate,
 
   void InitFromOptions(const mate::Dictionary& options);
 
+  // renderer triggered fullscreen mode
+  void ToggleFullscreen(bool enter_fullscreen);
```

Comment:
The declaration of this method should be placed near the `UpdateDraggableRegions`. Also the commend should be a full sentence (has first char capitalized and owns a period in the end).